xmake.lua

```diff
@@ -63,4 +63,5 @@ elseif is_mode("release") then
 end
 
  -- include xmake projects
-includes("**/xmake.lua")
+includes("Core/xmake.lua")
+includes("Samples/**/xmake.lua")
```

Comment:
Doesn't the first include auto include every .xmake file? Remove the second include

Reply:
It breaks the workflow, here is the similar issue https://github.com/xmake-io/xmake/issues/3448

Reply:
That's fine then :)